release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -42,6 +42,40 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Language
 
+##### Support for Module level complex variables
+
+Variable declarations with list, mapping, and error binding patterns are now allowed at module level. Unlike simple variables, these variables must be initialized in the declaration.
+
+Complex variable declarations cannot contain `isolated` or `configurable` qualifier.
```

Comment:
```suggestion
Complex variable declarations cannot contain `isolated` or `configurable` qualifiers.
```

Reply:
Or in-line with what @praneesha suggested in https://github.com/ballerina-platform/ballerina-release/pull/102#discussion_r589303917

```suggestion
Complex variable declarations cannot contain an `isolated` or `configurable` qualifier.
```